_posts/2021-03-23-connector-build.md

```diff
@@ -0,0 +1,223 @@
+Artifact `type` could be either `zip`, `tgz` or `jar` and of course has to be same as artifact available on `url`.
+`sha512sum` is optional property, which is used for validation of downloaded artifact by Strimzi, if it's specified.
+
+## Quick example
+
+Now lets go through quick example how to make it working in your cluster. 
+At first, you have to have Strimzi and Kafka cluster up and running. 
+Then you can create `KafkaConnect` with the following configuration (just change registry and organization in image name and secret):
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+spec:
+  replicas: 1
+  bootstrapServers: my-cluster-kafka-bootstrap:9092
+  config:
+    group.id: my-connect-cluster
+    offset.storage.topic: my-connect-cluster-offsets
+    config.storage.topic: my-connect-cluster-configs
+    status.storage.topic: my-connect-cluster-status
+    key.converter: org.apache.kafka.connect.json.JsonConverter
+    value.converter: org.apache.kafka.connect.json.JsonConverter
+    key.converter.schemas.enable: true
+    value.converter.schemas.enable: true
+    config.storage.replication.factor: 3
+    offset.storage.replication.factor: 3
+    status.storage.replication.factor: 3
+  build:
+    output:
+      type: docker
+      image: my-reg.io/my-org/my-connect-cluster:latest
+      pushSecret: my-registry-credentials
+    plugins: 
+      - name: camel-timer
+        artifacts:
+          - type: tgz
+            url: https://repo.maven.apache.org/maven2/org/apache/camel/kafkaconnector/camel-timer-kafka-connector/0.8.0/camel-timer-kafka-connector-0.8.0-package.tar.gz
+            sha512sum: c0102700ae176b1ef078bdb86d640935a1333865e58b870f6b0ef3d9dad067221122aee6bf52852dad209a7c2051f359e0434fc7fbd783fb50e21707c7577ef9
+  template:
+    pod:
+      imagePullSecrets:
+        - name: my-registry-credentials
+```
+
+Strimzi will spin-up Kafka Connect build pod and when build is finished, regular Kafka Connect pod will be created.
+
+```shell
+NAME                                          READY   STATUS      RESTARTS   AGE
+my-cluster-entity-operator-7d7f49cbc-b47cw    3/3     Running     0          4m10s
+my-cluster-kafka-0                            1/1     Running     0          4m43s
+my-cluster-kafka-1                            1/1     Running     0          4m43s
+my-cluster-kafka-2                            1/1     Running     0          4m43s
+my-cluster-zookeeper-0                        1/1     Running     0          5m14s
+my-cluster-zookeeper-1                        1/1     Running     0          5m14s
+my-cluster-zookeeper-2                        1/1     Running     0          5m14s
+my-connect-cluster-connect-69bc4bc47c-tvjzh   1/1     Running     0          74s
+my-connect-cluster-connect-build-1-build      0/1     Completed   0          2m2s
```

Comment:
You cheated here ... this example is from OCP.  The pod name will be different on Kube:

```suggestion
my-connect-cluster-connect-build              0/1     Completed   0          2m2s
```